flocker/node/agents/blockdevice_manager.py

```diff
@@ -0,0 +1,175 @@
+# -*- test-case-name: flocker.node.agents.test.test_blockdevice_manager -*-
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+This module implements interactions between the OS pertaining to block devices.
```

Comment:
I'd suggest dropping "This module implements" and just starting with "Interactions".